lib/components/DateTime/DatePicker.tsx

```diff
@@ -268,13 +269,11 @@ export class DatePicker extends React.Component<DatePickerProps, Partial<DatePic
         }
 
         if (valid) {
-            const hasVal = !!this.state.initialValue;
             let parsed = new MethodDate(
                 this.props.localTimezone,
-                year, month - 1, date,
-                hasVal ? this.state.initialValue.hours : 0,
```

Comment:
is the initial value passed in through another way now

Reply:
the initial value is still coming through the same way, but because this is a date picker and not a date time picker, it should ignore time